--- app/build.gradle.kts
+++ app/build.gradle.kts
@@ -77,6 +77,10 @@
   implementation(libs.ktor.client.core)
   implementation(libs.ktor.client.plugins)
   implementation(libs.ktor.client.android)
+  implementation(libs.ktor.client.cio)
+  implementation(libs.ktor.client.content.negotiation)
+  implementation(libs.ktor.serialization.kotlinx.json)
+  implementation(libs.ktor.client.timeout)
   implementation(platform(libs.androidx.compose.bom))
   implementation(platform(libs.firebase.bom))
   // implementation(libs.accompanist.permissions)
